pajbot/bot.py

```diff
@@ -833,6 +834,10 @@ def on_clearchat(self, chatconn, event):
                 # this sets timeout_end to None
                 user.timed_out = False
 
+    def on_irc_connect(self):
+        for p in self.phrases["welcome"]:
+            self.privmsg(p.format(nickname=self.nickname, version=self.version_long))
+
```

Comment:
This method now gets called by `IRCManager`, previously we used `on_connect` which was passed into the `Reactor(self.on_connect)`